TrainingsSite/style.css

```diff
@@ -0,0 +1,141 @@
+body{ margin: 0; 
+       background-color: #f0f2f3; 
+       font-family: Roboto, serif;
+}
+header{ background-color: black; 
+        color: white; 
+        height: 35px; 
+        line-height: 20px;
+        font-size: 20px;
+        padding-left: 20px;
+}
+span{ padding-left: 35px;
+}
+a{ text-decoration: none;
+   font-size: 15px;
+   color: white;
+   padding-left: 10px;
+}
+section{ height: calc(100vh - 35px);
+         background-image: url(./pexels-photo-583846.jpeg);
+         display: flex;
+         align-items: center;
+         background-repeat: no-repeat;
+         background-position: center center;
+         background-attachment: fixed;
+         background-size: cover;
+}
+h3{ color: white;
+    font-size: 55px;
+    padding-left: 150px;
+}
+@media (max-width: 800px){
+    section{ 
+      align-items: flex-start;
+      justify-content: flex-start;
+      display: flex;
+      justify-content: center;}
+    h3{
+       padding-left: 0; 
+       text-align: center;}
+}
+@media (max-width: 320px){
+    section{ 
+      justify-content: flex-start;
+      display: flex;
+      justify-content: center;
+      background-image: url(./pexels-photo-583846.jpg);}
+    h3{
+      text-align: center;}
+}
+main{ padding: 0 20px 20px;
+}
+p{ text-align: justify;
+   text-indent: 30px;
+}
+h2{ text-transform: uppercase; 
+    text-align: center;
+    color: #252525;
+}
+.menu{ padding-bottom: 25px;
+}
+.clear{ clear: both;
+}
+div > div{ background-color: white; 
+           color: #8abbe6;
+           height: 40px;
+           line-height: 40px;
+           width: 21%;
+           float: left;
+           padding-left: 20px;
+           margin: 5px;
+           min-width: 255px;
+}
+div > div:hover{ box-shadow: -1px 6px 16px 1px rgba(5, 5, 5, 0.925);
+                 transition: all 1s;
+                 text-decoration: underline;
+}
+table{ border-collapse: collapse;
+       border-radius: 5px;
+       background-color: white; 
+       width: 100%;
+       table-layout: fixed;
+       min-width: 800px;
+}
+.table_scroll{ overflow: scroll;
+}
+.bottom_line{ border-bottom: solid #868b85 1px;
+}
+.left_cell  div:nth-child(-n+3){
+    padding: 5px 5px 5px 15px;
+} 
+.left_cell div:first-child{ 
+    color: #4c96d8; 
+    cursor: pointer;
+}     
+.left_cell div:first-child:hover{ 
+    color: #151d3d;
+}
+.left_cell div:last-child{
+    font-size: 15px;
+}
+.middle_cell div:nth-child(-n+3){
+    padding: 5px;
+}
+.button{
+    width: 190px;
+    padding-right: 15px;
+}
+.button div:first-child{ 
```

Comment:
.button div:first-child and .button div:last-child have the same code
Please create class .button with common css prop.

Reply:
correct